### PR TITLE
Workaround for syntax highlighting issue in GitHub

### PR DESCRIPTION
**Pull request by neikeq**

### CODE REVIEW

File: modules/mono/editor/bindings_generator.cpp [hunk 1]
```diff
@@ -100,7 +100,7 @@
 
 #define BINDINGS_GENERATOR_VERSION UINT32_C(8)
 
-const char *BindingsGenerator::TypeInterface::DEFAULT_VARARG_C_IN = "\t%0 %1_in = %1;\n";
+const char*BindingsGenerator::TypeInterface::DEFAULT_VARARG_C_IN("\t%0 %1_in = %1;\n");
```
Comment (akien-mga) [inline modules/mono/editor/bindings_generator.cpp]: Won't that fail `clang-format`?

Comment (neikeq) [reply modules/mono/editor/bindings_generator.cpp]: Not for the one I have. Let's see what travis has to say.

**Comment by neikeq** [comment]
BTW, any idea were to report this syntax highlighting issue? In the linguist repo maybe?

**Comment by akien-mga** [comment]
> BTW, any idea were to report this syntax highlighting issue? In the linguist repo maybe?

Not directly, see: https://github.com/github/linguist#theres-a-problem-with-the-syntax-highlighting-of-a-file

The provider for C++ is https://github.com/textmate/c.tmbundle

**Comment by neikeq** [comment]
Thanks, looks the same as textmate/c.tmbundle#51

File: modules/mono/editor/bindings_generator.cpp (hunk 1 above)
Comment (neikeq) [reply modules/mono/editor/bindings_generator.cpp]: Never mind, I just realized you were referring to `char*`. Not sure how that happened.

Comment (neikeq) [reply modules/mono/editor/bindings_generator.cpp]: Fixed.

**Comment by hpvb** [comment]
Cherry-picked for 3.1.1.

**Pull request merged**